layouts/partials/fragments/hero.html

```diff
@@ -9,20 +9,17 @@
     {{/* Do not change the following snippet */}}
     {{/* Code is duplicated throughout the code */}}
     {{- $.root.Scratch.Set "image" (printf "images/%s" $image) -}}
-
     {{/* Page specific resource */}}
     {{- $location := (printf "%s/%s" $.fallthrough.page_path $image) -}}
     {{- if (fileExists (printf "content/%s" $location)) -}}
       {{/* special case index: trim _index/ from url */}}
-      {{- $location := strings.TrimPrefix "_index/" $location -}}
```

Comment:
Why remove this code snippet only in hero? Due to WIP or a different reason?

Reply:
Will look into the why later today hopefully.

Reply:
I think you added the `_index` part since the _index directory was a hack since we were using `url` to change it's address. Removing it was required so the image would be displayed. Using a list layout there's not much need for the url to be changed. Using `getPage` image is located inside _index directory and it need to be placed there.